pkg/cidata/template.go

```diff
@@ -39,6 +39,7 @@ type TemplateArgs struct {
 	SlirpNICName string
 	SlirpGateway string
 	SlirpDNS     string
+	LocalPort    int // host port for ssh (tcp) and dns (udp)
```

Comment:
`SSHLocalPort`, and `UDPDNSLocalPort`

Reply:
Should I still add both? We only need `UPDDNSLocalPort` in the template (once I figure out how to do the iptables rule).

If `iptables` doesn't work out, then we may not need it at all, but just create a little forwarder inside the guest-agent. But using iptables should be more efficient.